### ISSUE TITLE
Forecast weather for the time of the event

### ISSUE THREAD

**Issue by sdaveas** [sdaveas/tg-weather-bot#3]
Instead of getting the current weather conditions at the time of execution, it'd be better UX to get the forecast for the time of the event. Event's start time can be found at [`event['start']`](https://github.com/sdaveas/tg-weather-bot/blob/6dae8505402bd3cee2924f8a8eea97c298d1d769/forecast.py#L60) and can be passed to [`send_weather_forecast(...)`](https://github.com/sdaveas/tg-weather-bot/blob/master/forecast.py#L32C6-L32C6).

**Comment by sdaveas** [comment]
use https://openweathermap.org/forecast5

```
api.openweathermap.org/data/2.5/forecast?lat={lat}&lon={lon}&appid=[{API key}](https://home.openweathermap.org/api_keys)
```

response includes forecast per 3 hours for 5 days
```json
  {
      "dt": 1698883200,
      "main": {
        "temp": 293.81,
        "feels_like": 293.89,
        "temp_min": 293.23,
        "temp_max": 293.81,
        "pressure": 1013,
        "sea_level": 1013,
        "grnd_level": 990,
        "humidity": 75,
        "temp_kf": 0.58
      },
      "weather": [
        {
          "id": 804,
          "main": "Clouds",
          "description": "overcast clouds",
          "icon": "04n"
        }
      ],
      "clouds": {
        "all": 100
      },
      "wind": {
        "speed": 2.06,
        "deg": 245,
        "gust": 2.33
      },
      "visibility": 10000,
      "pop": 0,
      "sys": {
        "pod": "n"
      },
      "dt_txt": "2023-11-02 00:00:00"
    },
    {
      "dt": 1698894000,
      "main": {
        "temp": 293.44,
        "feels_like": 293.43,
        "temp_min": 292.7,
        "temp_max": 293.44,
        "pressure": 1012,
        "sea_level": 1012,
        "grnd_level": 989,
        "humidity": 73,
        "temp_kf": 0.74
      },
      "weather": [
        {
          "id": 804,
          "main": "Clouds",
          "description": "overcast clouds",
          "icon": "04n"
        }
      ],
      "clouds": {
        "all": 100
      },
      "wind": {
        "speed": 2.34,
        "deg": 247,
        "gust": 2.81
      },
      "visibility": 10000,
      "pop": 0,
      "sys": {
        "pod": "n"
      },
      "dt_txt": "2023-11-02 03:00:00"
    },
```

**Comment by sdaveas** [comment]
addressed in #7
